torch/fx/experimental/symbolic_shapes.py

```diff
@@ -565,6 +568,7 @@ def _maybe_evaluate_static(self, expr: "sympy.Expr") -> "Optional[sympy.Expr]":
         new_shape_env = {
             k: sympy.Symbol(f"shape_{idx}", positive=True, integer=True) + 1
```

Comment:
Why the + 1 here?

Reply:
Sympy doesn't have > 1 assumption; it only has > 0 assumption. So we simulate it by "decrementing" the variables